cmake/config/CMakeLists.txt

```diff
@@ -159,6 +159,7 @@ CONFIGURE_FILE(
 INSTALL(FILES
   ${CMAKE_CURRENT_BINARY_DIR}/${DEAL_II_PROJECT_CONFIG_NAME}Config.cmake
   ${CMAKE_CURRENT_BINARY_DIR}/${DEAL_II_PROJECT_CONFIG_NAME}ConfigVersion.cmake
+  ${CMAKE_CURRENT_BINARY_DIR}/deal.IIFeatureConfig.cmake
```

Comment:
Please set to `${CMAKE_CURRENT_BINAR_DIR}/${DEAL_II_PROJECT_CONFIG_NAME}FeatureConfig.cmake`